src/index.html

```diff
@@ -11,7 +11,203 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      class="form"
+      method="post"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <label for="surname">Surname:</label>
+
+        <input
+          id="surname"
+          type="text"
+          name="surname"
+          autocomplete="off"
+          class="input"
+        >
+
+        <br>
```

Comment:
Remove <br> tags, you can achive this by grouping form-fiels into block element like this:
```suggestion
        <div class="form-field">
          <label for="name">Name:</label>

          <input
            type="text"
            name="name"
            id="name"
            autocomplete="off"
            required
          >
        </div>
```

When you wrap label and input into div, it will take 100% width and make newline automatically for next elements. Wrapp all fields like this and remove <br> :)